poetry.lock

```diff
@@ -3241,6 +3251,127 @@ files = [
     {file = "tomlkit-0.12.5.tar.gz", hash = "sha256:eef34fba39834d4d6b73c9ba7f3e4d1c417a4e56f89a7e96e090dd0d24b8fb3c"},
 ]
 
+[[package]]
+name = "torch"
+version = "2.2.2"
```

Comment:
So x86_64 users will be stuck on version 2.2.2 forever? Or more generally, do we expect python projects to publish wheels for both platform? Or will they just bother with arm64 from now on? 😕